### PR TITLE
Build, version, sign, and upload through Travis

### PR DESCRIPTION
**Pull request by kennsippell**
Currently we build the release-ready version of the android app manually. This change introduces a Travis job which builds and publishes to GitHub (not yet publishing to App Store).
#78 

It also splits our single large APK into two smaller ABI-specific APK files.
#79 

A sample of the script's output is available here: https://github.com/medic/medic-android/releases/tag/v0.4.25 and is currently being tested by the MoH-Zanzibar team (tracked in https://github.com/medic/medic-projects/issues/6353).

* As part of this PR, I've set several environment variables through the Travis UI for the `medic-android` repo. 
* All encrypted keys in this PR are associated with the GitHub user `medic-ci` which now has write access to the repositories.
* Once this PR is approved, I'll replicate this process for `medic-gateway` and `medic-collect` repos.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,23 +1,39 @@
 language: android
+dist: trusty
```
Comment (garethbowen) [inline .travis.yml]: This is a 5 year old release. If this is needed, please add a comment.

File: README.md [hunk 2]
```diff
@@ -28,28 +28,17 @@ To add a new brand:
...
-	_initial load_: upload the .apk file to the APK page in the play store dev console
+1. Create a git tag starting with `v`, e.g. `v1.2.3` and push the tag to GitHub. Travis CI will build the release and push a signed, versioned, release-ready bundle to https://github.com/medic/medic-android/releases.
 
-	_updates_: trigger the job `medic-android-publish`, selecting the version, release track, and branding you wish to publish
+Then, for each brand you'd like to publish:
 
-1. repeat the previous step if you want to publish the app for multiple brands
+1. Download all APKs for the flavor you'd like to release from [GitHub Releases](https://github.com/medic/medic-android/releases).
+1. Navigate to the Google Play Console. `Release management` > `App releases`.
+1. Upload all APKs
```
Comment (garethbowen) [inline README.md]: Awesome to get rid of Jenkins!

File: build.gradle [hunk 3]
```diff
@@ -107,7 +112,14 @@ android {
...
 			buildConfigField "String", "SIMPRINTS_MODULE_ID", '"Medic Module ID"'
-		}
+    }
+
+    // Every APK requires a unique version code. 
+    // So when compiling multiple APKS for the different ABIs, use the first digit
+    variant.outputs.each { output ->
+      def versionAugmentation = (output.getFilter(OutputFile.ABI) == 'arm64-v8a') ? 1 : 0;
+      output.versionCodeOverride = variant.versionCode * 10 + versionAugmentation
+    }
```
Comment (garethbowen) [inline build.gradle]: Switching between tabs and spaces is breaking my brain. Could you make this whole file consistent, one way or the other?

File: .travis.yml (hunk 1 above)
Comment (kennsippell) [reply .travis.yml]: `Android builds are only supported on our Trusty image at this time hence you’ll need to explicitely specify dist: trusty in your .travis.yml file.`

https://docs.travis-ci.com/user/languages/android/

File: build.gradle (hunk 3 above)
Comment (kennsippell) [reply build.gradle]: So updated

File: README.md (hunk 2 above)
Comment (kennsippell) [reply README.md]: Wahoo!

**Pull request merged**